ibis/tests/all/test_vectorized_udf.py

```diff
@@ -5,14 +5,13 @@
 from ibis.udf.vectorized import elementwise
 
 
-@elementwise(input_type=[dt.double], output_type=dt.double)
-def add_one(s):
-    return s + 1
-
-
 @pytest.mark.only_on_backends([Pandas, PySpark])
 @pytest.mark.xfail_unsupported
 def test_elementwise_udf(backend, alltypes, df):
+    @elementwise(input_type=[dt.double], output_type=dt.double)
```

Comment:
Why move inside?

Reply:
Oops, I had moved this in when I created the test that checks that `output_type` works the same (I've removed that test now though). I can revert this

Reply:
Yeah let's revert unnecessary changes

Reply:
I added a test to check that there's an error when `output_type` is a list and it has an inline `add_one` UDF, so I'm keeping this change (moving global `add_one` to be inline inside `test_elementwise_udf`)